analysis/warriorfury/src/modules/talents/Siegebreaker.tsx

```diff
@@ -28,7 +28,9 @@ class Siegebreaker extends Analyzer {
 
   constructor(options: Options) {
     super(options);
-    this.active = this.selectedCombatant.hasTalent(SPELLS.SIEGEBREAKER_TALENT.id);
+    this.active =
```

Comment:
Why is this legendary required to show the stat?

Reply:
Is it safe to assume a player would never use Deathmaker and choose the Siegebreaker talent. They both apply the same debuff spell id